src/RemoteInboxNotifications/RuleEvaluator.php

```diff
@@ -63,6 +63,10 @@ public function evaluate( $rules, $stored_state = null, $logger_args = array() )
 		}
 
 		foreach ( $rules as $rule ) {
+			if ( !is_object( $rule ) ) {
```

Comment:
I'm seeing failed `Lint the PHP` job. You can see the errors on https://github.com/woocommerce/woocommerce-admin/runs/4876120052?check_suite_focus=true.

I'm not sure how it passed the lint pre-hook locally.

You can fix the style automatically by running the following command.

`./vendor/bin/phpcbf --standard=phpcs.xml.dist src/RemoteInboxNotifications/RuleEvaluator.php`

Reply:
I did just a c&p from the Github UI...

Reply:
We still need to fix the style errors as it is a required task to merge.

```
 66 | ERROR | [x] Expected 1 space after "!"; 0 found
 69 | ERROR | [x] Whitespace found at end of line
```